src/js/components/TextInput/TextInput.js

```diff
@@ -121,6 +122,22 @@ const TextInput = forwardRef(
 
     const [focus, setFocus] = useState();
     const [showDrop, setShowDrop] = useState(false);
+    const [arrowDownInitiated, setArrowDownInitiated] = useState(false);
+
+    const usingKeyboard = useKeyboard();
+    useEffect(() => {
+      const handleKeyDown = (event) => {
+        if (event.key === 'ArrowDown') {
+          setArrowDownInitiated(true);
```

Comment:
I'm not really seeing how this is reliable. The first time an ArrowDown happens the arrowDownInitiated is set true and never changes after that. Nothing sets it back to false. Does that mean we never expect to be able to get back to a state where this should be false?

Reply:
good call out I would set back the state to false when the drop closes but going to do a deeper look into this with Bill